### PR TITLE
Update meta description

### PR DESCRIPTION
**Pull request by metinbaris**
Updates meta description and meta title.

[2375](https://github.com/yunity/karrot-frontend/issues/2375)

✅   static values changed
✅  asked @nicksellen  for code review
✅ joined [chat.foodsaving.world/channel/karrot-dev](https://chat.foodsaving.world/channel/karrot-dev)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
   "private": true,
   "name": "karrot-frontend",
-  "productName": "Karrot",
+  "siteName": "Karrot",
```
Comment (nicksellen) [inline package.json:4]: This should stay as `productName`, it's a field specified by quasar here https://quasar.dev/quasar-cli/quasar-conf-js#property-build

> productName	String	Default value is taken from package.json > productName field.

File: package.json [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
   "version": "9.4.0",
-  "description": "Modern website for organization of foodsaving groups worldwide",
+  "title": "Karrot - Start a group, become a community",
```
Comment (nicksellen) [inline package.json:6]: `title` field isn't defined as part of [`package.json` spec](https://docs.npmjs.com/cli/v7/configuring-npm/package-json), or [quasars spec](https://quasar.dev/quasar-cli/quasar-conf-js#property-build).

Seems the best way to define extra properties for using in the html template is like this:
https://quasar.dev/quasar-cli/quasar-conf-js#property-htmlvariables

File: src/index.template.html [hunk 3]
```diff
@@ -1,17 +1,18 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= productName %></title>
-    <meta name="description" content="<%= productDescription %>">
+    <title><%= title %></title>
```
Comment (nicksellen) [inline src/index.template.html:4]: I think `productName` is still OK for this part - once the app loads we [set the title dynamically](https://github.com/yunity/karrot-frontend/blob/4b69cf31d55fac6fba3da27c1f8f9826b95cbd76/src/base/datastore/routerPlugin.js#L107)

File: src/index.template.html [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
+    <meta name="description" content="<%= description %>">
```
Comment (nicksellen) [inline src/index.template.html:5]: This can stay as `productDescription` I think? Which gets it from `package.json` description field already.

**Review by nicksellen — CHANGES_REQUESTED** [review]
Thanks! Almost there :)

Aside from the few changes I suggested in comments, can you remove  `node_modules/@quasar/app/lib/quasar-conf-file.js` from your changes too, that's part of the libraries we use, not our code. Maybe you added it by accident?

If you need more support with any of those things, feel free to ping me here, or in https://chat.karrot.world :+1:

File: package.json (hunk 1 above)
Comment (metinbaris) [reply package.json:4]: Haha need to read documentation more before commits maybe :D

File: package.json (hunk 2 above)
Comment (metinbaris) [reply package.json]: Hey @nicksellen  thanks for the review, think that i just added a new parsing to quasar instead of using its own function

**Comment by nicksellen** [comment]
Looks good, it's working as intended from https://github.com/yunity/karrot-frontend/issues/2375 now :)

Just see that the linting failed, if you click the "Details" above, it'll send you over to https://app.circleci.com/pipelines/github/yunity/karrot-frontend/2342/workflows/6e00400f-dac4-48c4-8245-adfe806c7afc/jobs/33145 where it shows two linting issues.

I think the smoothest way is making sure you have the linting tool (eslint) integrated into your editor (e.g. I use https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint in vscode). But you can also run it manually on that file with:

```
npx eslint quasar.conf.js 

/code/karrot/karrot-frontend/quasar.conf.js
  64:5   error  Expected space or tab after '//' in comment  spaced-comment
  66:58  error  Missing trailing comma                       comma-dangle

✖ 2 problems (2 errors, 0 warnings)
  2 errors and 0 warnings potentially fixable with the `--fix` option.
```

**Comment by metinbaris** [comment]
@nicksellen seems that the errors are gone this time 💯

**Comment by nicksellen** [comment]
The code looks good now!

Just before I approve, could you fill out the top comment in this PR a bit more? Partly as good practise, but it also helps if/when we are looking back at historic PRs to understand the context:

- I noticed the link to the issue on the first line is using the example issue still: _Closes ___ (e.g. #230)_
- could have a brief description instead of the deafult text: _(briefly describe what this PR is about)_
- check the relevant checkboxes in the checklist - one that is important is adding a changelog entry (for the testing part you could perhaps explain how you tried out it and verified it seems ok? but don't worry too much about that for this change)

Sorry, I should have mentioned this before. Thanks!

**Comment by nicksellen** [comment]
So close!

> one that is important is adding a changelog entry

Could add the changelog entry? You can see how the existing ones look here --> https://github.com/yunity/karrot-frontend/blob/master/CHANGELOG.md - this helps us keep track of the changes over time in one place, and becomes part of the release notes/messages.

File: CHANGELOG.md [hunk 5]
```diff
@@ -22,6 +22,12 @@ Please document your changes in this format:
 
 ## [Unreleased]
 
+## [9.4.1] - 2021-07-21
```
Comment (nicksellen) [inline CHANGELOG.md:25]: The release version and date is only added when we actually do a release, for now your changes will be "Unreleased", so they can stay in that section, see https://github.com/yunity/karrot-frontend/blob/master/CHANGELOG.md#unreleased for how that looks for merged PRs.

Comment (metinbaris) [reply CHANGELOG.md]: Perfect, no need for dates

**Review by nicksellen — APPROVED** [review]
Super! It's done :star:

**Pull request merged**